.env.example

```
@@ -1 +1,2 @@
-VITE_GOOGLE_TAG=
\ No newline at end of file
+VITE_GOOGLE_TAG=
+VITE_GOOGLE_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)